data/migrations/20200116153744_create-reactions-table.js

```diff
@@ -0,0 +1,28 @@
+
+exports.up = function(knex) {
+  return knex.schema.createTable("reactions", table => {
+      table.increments();
+      table.string("highlighted_text", 400).notNullable();
+      table.string("emoji", 400).notNullable();
+      table
+        .integer("articleId")
+        .unsigned()
+        .notNullable()
+        .references("id")
+        .inTable("articles")
+        .onUpdate("CASCADE")
+        .onDelete("CASCADE");
+      table
+        .integer("userId")
```

Comment:
Add both an authorId column and a reactorId column. The former should help with queries for notifications while the latter is good for keeping track of what someone has highlighted